### ISSUE TITLE
Als Erfasser von Daten möchte ich Attribute der Klasse Dachöffnungen anpassen (362)

### ISSUE THREAD

**Issue by LLanfermann** [CAFM-Connect/CAFM-Connect#20]
-	Anzahl Auslösestellen RWA

**Comment by vlaregon** [comment]
Zur Anpassung aller Attribute in der Klasse siehe Issue #61

**Comment by hdammbfs** [comment]
Die Klasse Dachöffnungen dient zur Kontrolle bei Sicherheitsbegehungen. RWA und NRA sind gesondert erfasst.
Ich schlage als Atrrbut Art der Dachöffnung vor: 
Auswahlfeld: Öffnung für technische Anlage, Öffnung zur Belichtung, ...

**Comment by vlaregon** [comment]
Nein, die Klasse 362 Dachöffnungen schließt natürliche Rauch- und Wärmeabzugsanlagen mit ein (laut DIN 276: Dachfenster, Ausstiege und andere Dachöffnungen einschließlich Umrahmungen, Beschlägen, Antrieben, Lüftungselementen und sonstiger Einbauteile; natürliche Rauch- und Wärmeabzugsanlagen)
Lediglich die maschinellen RWAs werden unter der Klasse 431 (in CC-Katalog unter 431.90) beschrieben.
Deshalb sollen NRA entweder direkt in der Klasse 362 oder in einer neuen zu ergänzenden und untergeordneten Klasse (z.B. 362.10) definiert werden. Demnach kann man ebenso für die normalen Dachfenster oder -ausstiege eine Unterklasse ergänzen (z.B. 362.20) und mit zutreffenden Merkmalen beschreiben.
In Issue #61 habe ich bereits einen Vorschlag zu Merkmalen der Klasse NRA beschrieben.
